docs/readmes/howtos/network_probe.md

```diff
@@ -85,48 +92,26 @@ Edit /etc/magma/pipelined.yml
 
 - Enable li_mirror in static_services list
 - Set the following items,
-    - li_local_iface: eth2
+    - li_local_iface: gtp_br0
     - li_mirror_all: false
     - li_dst_iface: li_port
+- restart pipelined
 
 ### 2. Enable LiAgentD service in AGW
 
-Edit /etc/magma/liagentd.yml
+Copy nprobe.{pem,key} to /var/opt/magma/certs/, then edit /etc/magma/liagentd.yml
```

Comment:
We might want to use codestyle here for better readability, e.g. `nprobe.{pem,key}` and `/var/opt/magma/certs/`, etc.